Gemfile

```diff
@@ -25,9 +25,10 @@ gem 'haml', '>= 4.0.5', require: false
 gem 'kramdown', '~> 2.3', require: false
 gem 'liquid', '>= 3.0', require: false
 gem 'redcarpet', '>= 3.1', require: false
-gem 'sassc', '~> 2', require: false
+gem 'sass-embedded', '~> 1', require: false
 gem 'sinatra', '>= 2.0.0.beta2', require: false
 gem 'slim', '>= 2.0', require: false
+gem 'tilt', github: 'rtomayko/tilt'
```

Comment:
This tilt gem override is currently needed or we have to install one of sassc/sass gem. I have asked in https://github.com/rtomayko/tilt/pull/367 to see if we can get tilt to cut a new release so that it won't panic.